department-of-veterans-affairs/va.gov-cms
# VACMS-8019 Remove covid limiter from facility health service push.

## Description

Closes #8019
Closes #8042

## Testing done


## Screenshots


## QA steps

Will be coordinating test with lighthouse.  Can not test by itself.  and needs to be done locally not on tugboat.

- Coordinate with Lighthouse engineers on testing.
- When they are ready set the appropriate endpoint and key in settings local.php 
-  go to /admin/config/va-gov-post-api/facility-force-queue 
- select -VAMC facility services and click the "Force Queue Selected Items button.
- The sandbox environment for lighouthouse has a API rate limit of 60 per minute.   Go here /admin/config/post-api/config set the rate limit to match.   Not sure if their dev version has a different rate limit.
- /admin/config/post-api/queue here you should find ~13,000 items in the queue.
- Scroll down and run click the "Process Queue" button.   
- At this point, Adam from lighthouse should be able to confirm that he sees 6 new services.  See if he says the data looks good.
-  repeat queue sending as many times as is needed for them to confirm all looks good (they likely do not need to receive all 13,000)
- If you need to clear the queue to retry something, you can do it with `drush queue:delete post_api_queue`
- Whenever lighthouse says we are good release this... it can be merged.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
